### PR TITLE
added wandb login

### PR DESCRIPTION
**Pull request by leonhardhennig**

### CODE REVIEW

File: wandb_init.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+
+def prepare_wandb():
+    # if wandb is installed
+    if find_spec("wandb"):
+        import wandb
+        wandb_api_key = os.getenv("WANDB_API_KEY", None)
+        if wandb_api_key is not None:
+            print('Logging into wandb')
+            wandb.login(key=wandb_api_key)
```
Comment (ArneBinder) [inline wandb_init.py]: Is this method called in any way? Maybe we do just this:
```suggestion
# if wandb is installed
if find_spec("wandb"):
    import wandb
    wandb_api_key = os.getenv("WANDB_API_KEY", None)
    if wandb_api_key is not None:
        print('Logging into wandb')
        wandb.login(key=wandb_api_key)
```

File: activate_and_execute.sh [hunk 2]
```diff
@@ -24,6 +24,9 @@ if [ -f "$ENV_FILE" ]; then
     bash "$SCRIPT_DIR"/import_vars.sh "$ENV_FILE"
 fi
 
+# check wandb and login
```
Comment (ArneBinder) [inline activate_and_execute.sh]:
```suggestion
# Login to Weights & Biases, if the wandb python package is installed 
# and the environment variable WANDB_API_KEY is set.
```

**Review by ArneBinder — CHANGES_REQUESTED** [review]
some remarks

File: wandb_init.py (hunk 1 above)
Comment (leonhardhennig) [reply wandb_init.py]: i just copied the code you sent me in Teams - I'm fine with changing it ;-)

Comment (ArneBinder) [reply wandb_init.py]: 👍 Ok, can you test if it also works for you with this proposed change?

Comment (leonhardhennig) [reply wandb_init.py]: yes, worked in multiple runs

Comment (ArneBinder) [reply wandb_init.py]: fixed in [d811634](https://github.com/DFKI-NLP/pegasus-bridle/pull/2/commits/d81163421cd07ad2ee652fcca4026e9425881485) - thanks :)

File: activate_and_execute.sh (hunk 2 above)
Comment (ArneBinder) [reply activate_and_execute.sh]: fixed in [d811634](https://github.com/DFKI-NLP/pegasus-bridle/pull/2/commits/d81163421cd07ad2ee652fcca4026e9425881485)

**Review by ArneBinder — APPROVED** [review]

**Pull request merged**